insights/parsers/sys_kernel.py

```diff
@@ -0,0 +1,38 @@
+"""
+SysKernel - file ``/proc/sys/kernel`` or ``/sys/kernel/``
+=========================================================
+
+The parsers included in this module are:
+
+SchedRTRuntime - File ``/proc/sys/kernel/sched_rt_runtime_us``
+--------------------------------------------------------------
+"""
+
+from insights import Parser, parser, LegacyItemAccess, get_active_lines
+from insights.parsers import ParseException
+from insights.specs import Specs
+
+
+@parser(Specs.sched_rt_runtime_us)
+class SchedRTRuntime(Parser, LegacyItemAccess):
+    """
+    Class for parsing the `/proc/sys/kernel/sched_rt_runtime_us` file.
+
+    Typical content of the file is::
+        950000
+
+    Examples:
+        >>> type(srt)
+        <class 'insights.parsers.sys_kernel.SchedRTRuntime'>
+        >>> srt.runtime_us
```

Comment:
For public `attributes` that need to be exposed to the user, it's necessary to describe them in the `Attributes` docstring.

See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google-style-python-docstrings